tests/test_search.py

```diff
@@ -236,10 +236,10 @@ def test_IndexBuilder():
         'envversion': '1.0',
         'filenames': ['filename1_1', 'filename1_2', 'filename2_1', 'filename2_2'],
         'objects': {
-            '': [
-                (0, 0, 1, '#anchor', 'objdispname1'),
-                (2, 1, 1, '#anchor', 'objdispname1'),
-            ]
+            '': {
+                0: (0, 0, 1, '#anchor', 'objdispname1'),
+                1: (2, 1, 1, '#anchor', 'objdispname1'),
```

Comment:
I think a JavaScript historian might inform us that in fact, `Array` was initially essentially syntactic sugar on top of the `Object` type (I don't know whether this is true or not).

What I've done here seems to inefficiently reinvent the past.  Let's see if some redundancy can be removed in a follow-up refactor here.